.changeset/lovely-days-cry.md

```diff
@@ -0,0 +1,7 @@
+---
+'@makeswift/runtime': patch
+---
+
+Add locale option to `getSitemap`. If a locale is using a domain-based localization, passing the locale to `getSitemap` will return the sitemap for that particular domain.
+
+For example, if in the site setting there is an `es` locale with the domain of `foo.es`, then passing `es` to `getSitemap` will return the sitemap for `foo.es`.
```

Comment:
'site setting" -> "site settings"

"with the domain of" -> "with a domain of"

Reply:
Updated